ark/phenotyping/som_utils.py

```diff
@@ -342,12 +511,16 @@ def cluster_pixels(fovs, base_dir, pre_dir='pixel_mat_preprocessed',
     misc_utils.verify_in_list(weights_columns=weights.columns.values,
```

Comment:
Looks like this check was already present in the helper function. Does this one check for order as well?

Reply:
No, even `verify_same_elements` doesn't. We should probably write a separate Python function that does check order or add that functionality to `verify_same_elements`.

Reply:
Yeah, you can add an `order` flag to verify_same_elements